### DISCUSSION TITLE
Basemaps Control Plugin

### DISCUSSION THREAD

**Discussion by ka7eh** [maplibre/maplibre-gl-js-docs#99] [Show and tell]
I needed a basemap switcher for MapLibre GL for a project, so I put something together and published it on npm.

If there's any interest, I can add it to the list of plugins here. Here's its link: https://github.com/ka7eh/maplibre-gl-basemaps.

Btw, thanks for all your efforts to keep this project going! :)

**Comment by HarelM** [comment]
Thanks for sharing this!
Another good place to report this is awesome-maplibre repo.
It's worth writing in the docs that this only supports raster sources I think :-)

**Reply by ka7eh** [reply]
Good to know about awesome-maplibre, and thanks for the reminder about raster sources! :)
